core/src/main/java/hudson/model/Computer.java

```diff
@@ -773,7 +773,7 @@ public String getCaption() {
     }
 
     public String getUrl() {
-        return "computer/" + Util.fullEncode(getName()) + "/";
+        return "manage/computer/" + Util.fullEncode(getName()) + "/";
```

Comment:
Nodes weren't linking correctly to the new Manage Jenkins parent url, this fixes that.

Reply:
While non-admin users can access `/manage/whatever` (e.g. `/manage/cli/`, `/manage/computer/`), this will add a breadcrumb they are not entitled to access directly (`/manage/` performs a permission check).

Reply:
Ah I see. I've updated the getter so it'll check if the user has permission and if so return the Manage link. Let me know if there's any issues with that approach 👍

Reply:
I cannot think of any off hand, but it feels kinda icky. At least it's not as bad as what we do for jobs in views 😁